src/joy.cpp

```diff
@@ -85,13 +97,70 @@ void Joy::StopMav() {
   command_msg_.normalized_throttle = 0;
 }
 
+/* Resets the mav back to origin */
+void Joy::ResetMav() 
+{
+	ROS_INFO("Mav position reset.");
+	ros::NodeHandle n;
+    geometry_msgs::Pose start_pose;
+	start_pose.position.x = 0.0;
+	start_pose.position.y = 0.0;
+	start_pose.position.z = 0.1;
```

Comment:
We might want to make these a parameter.  There are a few instances where we don't initialize the MAV at (0,0,0).  Definitely the defaults should be (0,0,0), but let's make it a parameter where it is reset to.